client/src/app/core/ui-services/csv-export.service.ts

```diff
@@ -164,20 +151,17 @@ export class CsvExportService {
     }
 
     /**
-     * Checks if a given input contains any of the characters defined in a list
-     * used for textseparators. The list is then returned without the 'special'
-     * characters, as they may not be used as text separator in this csv.
+     * Ensures, that the given string has escaped double quotes
+     * and no linebreak. The string itself will also be escaped by `double quotes`.
      *
-     * @param input any input to be sent to CSV
-     * @param tsList The list of special characters to check.
-     * @returns the cleaned CSV String list
+     * @param {string} input any input to be sent to CSV
+     * @returns {string} the cleaned string.
      */
-    public checkCsvTextSafety(input: string, tsList: string[]): string[] {
-        if (input === null || input === undefined) {
-            return tsList;
+    public checkCsvTextSafety(input: string): string {
+        if (!input) {
+            return '';
         }
-
-        return tsList.filter(char => input.indexOf(char) < 0);
+        return '"' + input.replace(/"/g, '""').replace(/(\r\n\t|\n|\r\t)/gm, ".") + '"';
```

Comment:
Okay, going the way of changing the exported strings. Are the more complicated ones (e.g. with quotes, html formatting, special characters, linebreaks) still save for re-import? (That's the most common use case I heard about)

Reply:
Yes, I've tested this by trying re-importing the motions exported before.